pages/app-layout/with-wizard-and-table.page.tsx

```diff
@@ -0,0 +1,223 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import {
+  AppLayout,
+  Box,
+  Button,
+  CollectionPreferences,
+  ColumnLayout,
+  Container,
+  Header,
+  Pagination,
+  SpaceBetween,
+  Table,
+  TextFilter,
+  Wizard,
+} from '~components';
+import ScreenshotArea from '../utils/screenshot-area';
+
+export default function () {
+  const [activeStepIndex, setActiveStepIndex] = useState(0);
+
+  return (
+    <ScreenshotArea gutters={false}>
+      <AppLayout
+        navigationHide={false}
+        content={
+          <Wizard
+            i18nStrings={{
+              stepNumberLabel: stepNumber => `Step ${stepNumber}`,
+              collapsedStepsLabel: (stepNumber, stepsCount) => `Step ${stepNumber} of ${stepsCount}`,
+              skipToButtonLabel: step => `Skip to ${step.title}`,
+              navigationAriaLabel: 'Steps',
+              cancelButton: 'Cancel',
+              previousButton: 'Previous',
+              nextButton: 'Next',
+              submitButton: 'Launch instance',
+              optional: 'optional',
+            }}
+            onNavigate={({ detail }) => setActiveStepIndex(detail.requestedStepIndex)}
+            activeStepIndex={activeStepIndex}
+            allowSkipTo={true}
+            steps={[
+              {
+                title: 'Add storage',
+                content: (
+                  <Box>
+                    <Table
+                      ariaLabels={{
+                        selectionGroupLabel: 'Items selection',
+                        allItemsSelectionLabel: ({ selectedItems }) =>
+                          `${selectedItems.length} ${selectedItems.length === 1 ? 'item' : 'items'} selected`,
+                        itemSelectionLabel: ({ selectedItems }, item) => {
+                          const isItemSelected = selectedItems.filter(i => i.name === item.name).length;
+                          return `${item.name} is ${isItemSelected ? '' : 'not'} selected`;
+                        },
+                      }}
+                      columnDefinitions={[
```

Comment:
there is some shared configs and generated items for tables in these files
https://github.com/cloudscape-design/components/blob/main/pages/table/shared-configs.tsx
https://github.com/cloudscape-design/components/blob/main/pages/table/generate-data.ts

Reply:
Excellent, thank you!